data/sql_updates/create_committee_history.sql

```diff
@@ -79,35 +67,33 @@ select distinct on (dcp.cmte_sk, cycle)
     dcp.cmte_custodian_title as custodian_name_title,
     dcp.cmte_custodian_zip as custodian_zip,
     -- Properties
-    dcp.cmte_email as email,
+    fec_yr.cmte_email as email,
     dcp.cmte_fax as fax,
-    dcp.cmte_web_url as website,
+    fec_yr.cmte_url as website,
     dcp.form_tp as form_type,
     dcp.leadership_pac as leadership_pac,
-    dcp.load_date as load_date,
```

Comment:
I think that date changed could be valuable if you wanted to sort and see committees that had recent updates. I think there might be value to finding the new and changing content for advanced data users.